tests/unit/helpers/utils.py

```diff
@@ -203,7 +207,7 @@ def foo_ref_attr(self, value):
             raise ValueError("can't reset foo_ref_attr attribute")
 
 
-def get_foo_buildmanager():
+def get_foo_buildmanager(dtype="int"):
```

Comment:
```suggestion
def get_foo_buildmanager(my_data_dtype="int"):
```
`dtype` is very ambiguous since there are multiple objects with dtypes.